### PR TITLE
Remove deprecated colors and add color palette 🎨

### PR DESCRIPTION
**Pull request by gustaveen**
## What

- Remove deprecated colors. They have all been removed from repos using brand colors.
- Add color palette to Storybook with brand guidelines.

## Why 

So we are aligned with brand colors.

https://user-images.githubusercontent.com/6661511/145245246-6eb26d9f-fab4-4ac4-b07d-9412c7e5f47c.mov

### CODE REVIEW

File: src/GlobalStyles.tsx [hunk 1]
```diff
@@ -51,6 +51,26 @@ export const globalStyles = css`
...
+    margin-bottom: 0.5rem;
+    letter-spacing: -0.02em;
+    font-size: 1.5rem;
+  }
+
+  p {
+    margin-top: 0;
+    color: ${colorsV3.gray700};
+    font-size: 1.125rem;
+  }
```
Comment (robinandeer) [inline src/GlobalStyles.tsx:72]: What do we need these for? Markdown content?

File: src/colors/colors.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  gap: 0.5rem;
+  margin-top: 2rem;
+  margin-bottom: 2.5rem;
+`
+
+const ColorSwatch = styled.div<{ color: string }>`
+  width: 110px;
+  padding-top: 92px;
+  text-align: center;
+  border: 1px solid #c7c7c7;
```
Comment (robinandeer) [inline src/colors/colors.stories.tsx:17]: haha guess we don't have that color in the palette? 😅

**Review by robinandeer — APPROVED** [review]
Very nice work!

File: src/colors/colors.stories.tsx (hunk 2 above)
Comment (gustaveen) [reply src/colors/colors.stories.tsx:17]: hehe yes, it needed to be a non theme color si the border wouldn't match any of the gray colors :) so it's for the sake of this visualisation only :)

File: src/GlobalStyles.tsx (hunk 1 above)
Comment (gustaveen) [reply src/GlobalStyles.tsx:72]: Ah yes, it's just for styling for the heading and text in the story to create that kind of style guide view :)

**Pull request merged**